spec/symbol-store-spec.coffee

```diff
@@ -100,6 +100,29 @@ describe 'SymbolStore', ->
       expect(occurrences[0].symbol.text).toBe 'abc'
       expect(occurrences[0].symbol.type).toBe 'newtype'
 
+    it "dont override description of built-in function", ->
```

Comment:
What do you think about rewording this as `doesn't override built-in suggestions with the symbols found in the buffer`?